### PR TITLE
Edit controllers

### PR DESCRIPTION
**Pull request by deathCodeDevelop**

### CODE REVIEW

File: WebUI/Areas/Admin/Views/Role/_ViewAll.cshtml [hunk 1]
```diff
@@ -28,24 +30,26 @@
...
+                                </form>
 
                                 <a style='margin-right:16px' asp-controller="Permission" asp-action="Index" asp-route-roleId="@role.Id" class="dropdown-item">
-                                    <i class="fas fa-wrench"></i>Manage Permissions
+                                    <i class="fas fa-wrench"></i> Керувати Дозволами
                                 </a>
-                            }
-                        </div>
+                            </div>
+                             }
```
Comment (ColdForeign) [inline WebUI/Areas/Admin/Views/Role/_ViewAll.cshtml:51]: Unnecessary parenthesis

File: WebUI/Areas/Admin/Views/User/_Create.cshtml [hunk 2]
```diff
@@ -1,53 +1,80 @@
...
-        <button type="submit" class="btn btn-default close-button" data-dismiss="modal">Cancel</button>
+        <button type="submit" class="btn btn-success">Зберегти</button>
+        <button type="submit" class="btn btn-default close-button" data-dismiss="modal">Відміна</button>
     </div>
 </form>
+}
+
 <script type="text/javascript" language=javascript>
     $.validator.unobtrusive.parse(document);
 </script>
```
Comment (ColdForeign) [inline WebUI/Areas/Admin/Views/User/_Create.cshtml:80]: No newline at end of file

Comment (deathCodeDevelop) [reply WebUI/Areas/Admin/Views/User/_Create.cshtml:80]: </script>
```

**Review by ColdForeign — CHANGES_REQUESTED** [review]

File: WebUI/Areas/Entities/Controllers/CommunityController.cs [hunk 3]
```diff
@@ -83,25 +89,32 @@ public async Task<JsonResult> OnPostCreateOrEdit(int id, CommunityViewModel bran
...
-                    //_notify.Error(response.Message);
+                    _notify.Error("Помилка створення");
                     return null;
                 }
             }
             else
             {
-                var html = await _viewRenderer.RenderViewToStringAsync("_CreateOrEdit", brand);
+                
+
```
[the remark this answers is not in the document]
Comment (deathCodeDevelop) [reply WebUI/Areas/Entities/Controllers/CommunityController.cs:99]:
```      
```

File: WebUI/Areas/Entities/Views/Worker/_ViewAll.cshtml [hunk 4]
```diff
@@ -40,4 +40,4 @@
 <script>
     $(document).ready(function () {
         $("#brandTable").DataTable();
-    });</script>
\ No newline at end of file
+    });</script>
```
[the remark this answers is not in the document]
Comment (deathCodeDevelop) [reply WebUI/Areas/Entities/Views/Worker/_ViewAll.cshtml:43]:
```suggestion
    });
    </script>
```

**Review by ColdForeign — APPROVED** [review]

**Review by ColdForeign — COMMENTED** [review]
good

**Pull request merged**